rrfs-test/testinput/rrfs_mpasjedi_2024052700_Ens3Dvar.yaml

```diff
@@ -45,7 +45,7 @@ cost function:
   analysis variables: *incvars
   background:
     state variables: [spechum,surface_pressure,temperature,uReconstructMeridional,uReconstructZonal,theta,rho,u,qv,pressure,landmask,xice,snowc,skintemp,ivgtyp,isltyp,snowh,vegfra,u10,v10,lai,smois,tslb,pressure_p,qc,qi,qg,qr,qs,cldfrac]
-    filename: data/restart.2024-05-27_00.00.00.nc
+    filename: data/mpasout.2024-05-27_00.00.00.nc
```

Comment:
I think we discussed this in another PR and we will keep file name as "restart.2024-05-27_00.00.00.nc"

Reply:
@ShunLiu-NOAA Could you point me to that PR and the reason why we want to use `restart` instead of `mpasout`? Thanks!

Reply:
Ah, I got it. I think you refer to my previous PR #161.  

With the update to mpasjedi.v3.0.1 and mpas.v8.2.2, we have to use `invariant.nc` and `mpasout.nc`.

For PR #161, Ting's testing did not find any differences between using `restart.nc` or `mpasout.nc` when using the same code base.

The bumploc file differences I saw should come from the different sabor/bump source code versions. It is worthy of further investigation, but not related to the change of `restart` to `mpasout`

Reply:
As discussed in PR #161, changing `restart` to `mapsout` does not bring any differences to the bumploc files.

And this change is needed to accommodate the new design in MPASv8.2.2 and mpasjedi v3.0.1